src/sage/manifolds/differentiable/diff_form.py

```diff
@@ -646,6 +652,9 @@ def hodge_dual(
           :class:`~sage.manifolds.differentiable.symplectic_form.SymplecticForm`.
           If none is provided, the ambient domain of ``self`` is supposed to be endowed
           with a default metric and this metric is then used.
+        - ``minus_eigenvalues_convention`` -- if `true`, a factor of `(-1)^s` is
+            introduced with `s` being the number of negative eigenvalues of the
```

Comment:
Would something like ``eigenvalues_convention='minus'`` be a better interface?

Reply:
Mhh, not sure. It's not really that this is a convention about the eigenvalues, but about the minus sign. So the most exact version would be `sign_convention = 'negative eigenvalues'`, but this is too verbose in my opinion.